forge-gui/res/cardsfolder/upcoming/nowhere_to_run.txt

```diff
@@ -0,0 +1,9 @@
+Name:Nowhere to Run
+ManaCost:1 B
+Types:Enchantment
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ When CARDNAME enters, target creature an opponent controls gets -3/-3 until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | NumAtt$ -3 | NumDef$ -3 | IsCurse$ True
+S:Mode$ IgnoreHexproof | Activator$ You | ValidEntity$ Creature.OppCtrl | Description$ Creatures your opponents control can be the targets of spells and abilities as though they didn't have hexproof. Ward abilities of those creatures don't trigger.

```

Comment:
Activator wrong

Reply:
just like `ValidCard$ Card` using `Activator$ Any` really serves no purpose
(other than script bloat + making the engine check for no reason)